### PR TITLE
Remove long obsolete DB-to-Endpoint mapping page

### PR DESCRIPTION
**Pull request by Simran-B**
The ability to restrict an endpoint to certain databases is last mentioned in 3.0. Was probably a 2.x feature.

https://github.com/arangodb/arangodb/blob/3.0/Documentation/Books/Manual/Administration/Configuration/Endpoint.md

### CODE REVIEW

File: 3.7/appendix-glossary.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ A single ArangoDB instance can handle multiple databases in parallel. By default
 
 Databases are physically stored in separate sub-directories underneath the database directory, which itself resides in the instance's data directory.
 
-Each database has its own sub-directory, named database-<database id>. The database directory contains sub-directories for the collections of the database, and a file named parameter.json. This file contains the database id and name.
+Each database has its own sub-directory, named `database-<database id>`. The database directory contains sub-directories for the collections of the database, and a file named parameter.json. This file contains the database id and name.
```
Comment (jsteemann) [inline 3.7/appendix-glossary.md:55]:
```suggestion
```

File: 3.7/http/database-notes-on-databases.md [hunk 2]
```diff
@@ -54,7 +53,7 @@ there will be at least one database which is named `_system`. Databases are
 physically stored in separate sub-directories underneath the database directory,
 which itself resides in the instance's data directory.
 
-Each database has its own sub-directory, named `database-<database id>`. The
+Each database has its own sub-directory, named ``database-<database id>``. The
 database directory contains sub-directories for the collections of the database,
 and a file named parameter.json. This file contains the database id and name.
```
Comment (jsteemann) [inline 3.7/http/database-notes-on-databases.md:58]:
```suggestion
```

File: 3.8/appendix-glossary.md [hunk 3]
```diff
@@ -52,7 +52,7 @@ A single ArangoDB instance can handle multiple databases in parallel. By default
 
 Databases are physically stored in separate sub-directories underneath the database directory, which itself resides in the instance's data directory.
 
-Each database has its own sub-directory, named database-<database id>. The database directory contains sub-directories for the collections of the database, and a file named parameter.json. This file contains the database id and name.
+Each database has its own sub-directory, named `database-<database id>`. The database directory contains sub-directories for the collections of the database, and a file named parameter.json. This file contains the database id and name.
```
Comment (jsteemann) [inline 3.8/appendix-glossary.md:55]:
```suggestion
```

File: 3.8/http/database-notes-on-databases.md [hunk 4]
```diff
@@ -54,7 +53,7 @@ there will be at least one database which is named `_system`. Databases are
 physically stored in separate sub-directories underneath the database directory,
 which itself resides in the instance's data directory.
 
-Each database has its own sub-directory, named `database-<database id>`. The
+Each database has its own sub-directory, named ``database-<database id>``. The
 database directory contains sub-directories for the collections of the database,
 and a file named parameter.json. This file contains the database id and name.
```
Comment (jsteemann) [inline 3.8/http/database-notes-on-databases.md:58]:
```suggestion
```

File: 3.9/appendix-glossary.md [hunk 5]
```diff
@@ -52,7 +52,7 @@ A single ArangoDB instance can handle multiple databases in parallel. By default
 
 Databases are physically stored in separate sub-directories underneath the database directory, which itself resides in the instance's data directory.
 
-Each database has its own sub-directory, named database-<database id>. The database directory contains sub-directories for the collections of the database, and a file named parameter.json. This file contains the database id and name.
+Each database has its own sub-directory, named `database-<database id>`. The database directory contains sub-directories for the collections of the database, and a file named parameter.json. This file contains the database id and name.
```
Comment (jsteemann) [inline 3.9/appendix-glossary.md:55]:
```suggestion
```

File: 3.9/http/database-notes-on-databases.md [hunk 6]
```diff
@@ -54,7 +53,7 @@ there will be at least one database which is named `_system`. Databases are
 physically stored in separate sub-directories underneath the database directory,
 which itself resides in the instance's data directory.
 
-Each database has its own sub-directory, named `database-<database id>`. The
+Each database has its own sub-directory, named ``database-<database id>``. The
 database directory contains sub-directories for the collections of the database,
 and a file named parameter.json. This file contains the database id and name.
```
Comment (jsteemann) [inline 3.9/http/database-notes-on-databases.md:58]:
```suggestion
```

**Comment by Simran-B** [comment]
@jsteemann WDYT about my suggestion?

Database Organization
---------------------

A single ArangoDB instance can handle multiple databases in parallel. By default,
there will be at least one database which is named `_system`. 

Data is physically stored in `.sst` files in a sub-directory `engine-rocksdb`
that resides in the instance's data directory. A single file can contain
documents of various collections and databases.

ArangoSearch stores data in database-specific directories underneath the
`databases` folder.

Foxx applications are also organized in database-specific directories but inside
the application path. The filesystem layout could look like this: ...

**Pull request merged**